.github/workflows/pr.yml

```diff
@@ -28,10 +28,10 @@ jobs:
         run: bun run lint
 
       - name: Build daemon
-        run : bun run postinstall
+        run : bun run postinstall 
```

Comment:
@jkone27 ...

Reply:
shall i remove the space right (yaml sensitivity), or is it ok?

Reply:
Remove the space, never ever introduce whitespaces for no reason.